analyzer/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     compile "org.apache.maven.resolver:maven-resolver-connector-basic:$mavenResolverVersion"
     compile "org.apache.maven.resolver:maven-resolver-transport-file:$mavenResolverVersion"
     compile "org.apache.maven.resolver:maven-resolver-transport-http:$mavenResolverVersion"
+    compile "org.apache.maven.resolver:maven-resolver-transport-wagon:$mavenResolverVersion"
```

Comment:
Does it really make sense to have a separate commit for this change, or should we squash?